config/functions

```diff
@@ -609,13 +609,16 @@ EOF
 }
 
 create_meson_conf_target() {
+  local linker="$(get_target_linker)"
```

Comment:
This makes sense. Does this mean that we have been using`bfd` for all meson:target builds? What is the impact of these builds now using gold or mold? (Well specifically gold as it is the default in the LE12 image?)

Reply:
Yes, I believe those have been using bfd all along.
I only tested mold for my allwinner board though. While that works just fine, I neither tested gold nor any other platform

Reply:
This one will need more testing then. I just noticed pixman on the 0.43.0 update is meson and it explicitly wanted bfd in configure, but with this change gold would be used. So let’s hold this commit back for LE12.